modules/ROOT/pages/cloudhub-impaired-worker.adoc

```diff
@@ -1,17 +1,59 @@
-= About Impaired Workers
+= Impaired Worker Monitoring 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-CloudHub performs constant health checks on the underlying virtual machines where Mule applications are running. A worker is impaired when there is a hardware or software failure that negatively impacts the running application deployed on the worker. Examples of hardware and software failures include network unavailability, memory issues, server hardware failures, file corruption, and so on.
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-fabric/deploy-to-runtime-fabric", title="Runtime Fabric"]
 
-When a worker is impaired, CloudHub automatically attempts to restart the worker and relaunch the application. If a retry is successful, the underlying worker and the Mule application are recovered without requiring any further action.
+CloudHub continuously performs health checks on the underlying virtual machines where Mule applications are running. 
+When a worker is impaired, CloudHub automatically attempts to restart the worker and relaunch applications.
 
-[NOTE]
+A worker is impaired when there is a hardware or software failure that negatively impacts the running application deployed on the worker.
+Hardware and software failures include network unavailability, memory issues, server hardware failures, and file corruption.
+
+When CloudHub restarts the impaired worker:
+
+* If a retry is successful, the underlying worker and app restart without any further action.
++
 It might take up to five minutes for a worker to restart.
+* If the application fails to start after five attempts, the status of the application is *Failed* or *Unrecoverable Runtime Error*.
++
+In this case, you must manually restart the application.
+
+The logs indicate when a worker is restarted automatically and any failures.
 
-If the application fails to start after five attempts, the status of the application is *Failed* or *Unrecoverable Runtime Error*. In this case, you must manually redeploy and restart the application.
+[NOTE]
+The automatic worker-restart functionality is different from the automatic application-restart functionality, which is configurable using the *Automatically restart application when not responding* option on the *Settings* page for the app. 
+CloudHub automatically attempts to restart impaired workers, regardless of the app setting. 
```

Comment:
This complicates the narrative flow further. So I have a new organizational suggestion:
1. If worker fails and causes app to fail, CloubHub uses auto restart on worker
2. Worker restarts (you can see in log)
3. If app is configured to auto restart and does, no further action (might take 5 min)
4. If app is not configured to auto restart or does not after 5 attempts, status changes (you can see in log)
5. You must manually restart app (and consider config auto if not?)

Reply:
Well, no. 3 and 4 don't apply. The app is restarted automatically when the worker restarts.